Update README (#456)
gamer-project/gamer: README.md

<h1 align="center">
🎮🖥️ <a href="">GAMER</a>: A <a href="">G</a>PU-accelerated 🌌👾<br>
<a href="">A</a>daptive <a href="">ME</a>sh <a href="">R</a>efinement code for astrophysics
</h1>

[![GitHub Release](https://img.shields.io/github/v/release/gamer-project/gamer)](https://github.com/gamer-project/gamer/releases)
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.15036956.svg)](https://doi.org/10.5281/zenodo.15036956)

> [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://github.com/gamer-project/gamer?tab=License-1-ov-file)

Author: Because our license contain `*Hilbert Curve implementation copyright 1998, Rice University`
which is not included in the BSD 3-Clause, I link it to our license page.

> [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://github.com/gamer-project/gamer?tab=License-1-ov-file)

Reviewer: I see, thanks for the explanation.

[![GitHub contributors](https://img.shields.io/github/contributors/gamer-project/gamer)](https://github.com/gamer-project/gamer/graphs/contributors)

> [![GitHub commit activity](https://img.shields.io/github/commit-activity/m/gamer-project/gamer)](https://github.com/gamer-project/gamer/pulse)

Reviewer 2: Got it. Thanks for the clarification.

> [![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.crossref.org%2Fworks%2F10.1088%2F0067-0049%2F186%2F2%2F457&query=%24.message.is-referenced-by-count&label=GAMER-1%20Citations)](https://iopscience.iop.org/article/10.1088/0067-0049/186/2/457)

Author: I get the number from the [Crossref REST API](https://api.crossref.org/swagger-ui/index.html). So it will not be the same as ADS.
Still, the number on [iopscience](https://iopscience.iop.org/article/10.1088/0067-0049/186/2/457) is 105.
Although the API of ADS is not public, I might have some easy workaround to get it automatically. Which metric source do you want?

> [![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.crossref.org%2Fworks%2F10.1088%2F0067-0049%2F186%2F2%2F457&query=%24.message.is-referenced-by-count&label=GAMER-1%20Citations)](https://iopscience.iop.org/article/10.1088/0067-0049/186/2/457)

Reviewer 2: Check if you can retrieve the citation counts directly from the journal websites:
- GAMER-1: [ApJS](https://iopscience.iop.org/article/10.1088/0067-0049/186/2/457)=105
- GAMER-2: [MNRAS](https://academic.oup.com/mnras/article/481/4/4815/5106358)=75

If not, feel free to use either Google Scholar or the Crossref REST API. Let’s avoid using ADS for now, as it tends to systematically underreport citation counts.

[![Dynamic JSON Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fapi.crossref.org%2Fworks%2F10.1093%2Fmnras%2Fsty2586&query=%24.message.is-referenced-by-count&label=GAMER-2%20Citations)](https://academic.oup.com/mnras/article/481/4/4815/5106358)

**GAMER** is a **G**PU-accelerated **A**daptive **ME**sh **R**efinement code that features
high performance, strong parallel scalability, and a wide range of physics modules.

> ## Getting Started & Documentation

Reviewer 2: ```suggestion
## Getting Started
```

> - **Quick Start Guide**: [Quick Start](https://github.com/gamer-project/gamer/wiki/Quick-Start)  
>   A step-by-step guide to quickly set up and run GAMER.
> - **Installation Instructions**: [Installation](https://github.com/gamer-project/gamer/wiki/Installation)  
>   Detailed instructions for installing GAMER on various platforms.
> - **Comprehensive Documentation**: [Wiki](https://github.com/gamer-project/gamer/wiki)  
>   Full documentation covering all aspects of GAMER, including usage, configuration, and advanced features.

Reviewer 2: ```suggestion
- [**Quick Start**](https://github.com/gamer-project/gamer/wiki/Quick-Start): Step-by-step guide to quickly set up and run GAMER.
- [**Installation Guide**](https://github.com/gamer-project/gamer/wiki/Installation): Detailed instructions for installing GAMER on various platforms.
- [**Full Documentation**](https://github.com/gamer-project/gamer/wiki): Comprehensive user guide covering usage, configuration, physics modules, and advanced features.
```
